aztec/src/main/kotlin/org/wordpress/aztec/util/Extensions.kt

```diff
@@ -56,3 +58,15 @@ fun ClipData.Item.coerceToHtmlText(parser: AztecParser): String {
 
     return text.toString()
 }
+
+/**
+ * Parses and applies the HTML 'style' attribute.
+ * @param output An [Editable] containing an [IAztecAttributedSpan] for processing.
+ * @param start The index where the [IAztecAttributedSpan] starts inside the [text]
+ */
+inline fun IAztecAttributedSpan.applyInlineStyleAttributes(output: Editable, start: Int) {
```

Comment:
I chose to make this an extension function to avoid making IAztecAttributedSpan abstract, and I made it inline so it would have access to the `attributes` variable.

Reply:
I'll swap this out and make it a function of IAztecAttributedSpan. Thank you for the feedback :)